tools/who_what_benchmark/whowhatbench/reranking_evaluator.py

```diff
@@ -164,11 +164,10 @@ def default_gen_answer(model, tokenizer, query, passages):
                 else:
                     scores = outputs.flatten()
                 scores = scipy.special.expit(scores)
-            sorted_scores = []
+            ordered_scores = []
             for index, (score, _) in enumerate(zip(scores, passages)):
-                sorted_scores.append(np.array([index, score.numpy()]))
-            sorted_scores.sort(key=lambda x: x[1], reverse=True)
-            return np.array(sorted_scores[:DEFAULT_TOP_K])
+                ordered_scores.append(np.array([index, score.numpy()]))
+            return np.array(ordered_scores[:DEFAULT_TOP_K])
```

Comment:
Do we need `DEFAULT_TOP_K` at all? We need all documents to be returned, maybe we don't need this slicing at all?
For genai reranker pipeline we have `top_n=3` by default, do we pass `top_n` for genai pipeline additionally?